plugins/inputs/exec/README.md

```diff
@@ -17,6 +17,10 @@ This plugin can be used to poll for custom metrics from any source.
     "/tmp/collect_*.sh"
   ]
 
+  ## Environment variables
+  ## eg: environment = ["LD_LIBRARY_PATH=/opt/custom/lib64:/usr/local/libs", "USERNAME=John Doe"]
+  environment = ["LD_LIBRARY_PATH=/opt/custom/lib64"]
```

Comment:
The value in the sample config should be the default and commented out. In this case, we don't want everyone's config exporting that library path.

```suggestion
  ## Array of "key=value" pairs to pass as environment variables
  ## e.g. "KEY=value", "USERNAME=John Doe",
  ## "LD_LIBRARY_PATH=/opt/custom/lib64:/usr/local/libs"
  # environment = []
```